components/homme/test/bubble/nh.nl

```diff
@@ -0,0 +1,68 @@
+!
+! namelist for planar nonhydrostatic gravity waves
+!_______________________________________________________________________
+&ctl_nl
+  nthreads          = 1
+  partmethod        = 4                         ! mesh parition method: 4 = space filling curve
+  topology          = "plane"                   ! mesh type: planar
+  geometry          = "plane"                   ! mesh type: planar
+  test_case         = "planar_rising_bubble"         		! test identifier
+  theta_hydrostatic_mode = .false.
+!  theta_hydrostatic_mode = .true.
+  transport_alg     = 0            
+  theta_advect_form = 1
+  vert_remap_q_alg  = 10
+  moisture          = 'dry'
+  ne_x              = 20                        ! number of elements in x-dir
+  ne_y              = 20                        ! number of elements in y-dir
+  qsize             = 0                         ! num tracer fields
+  nmax              = 12000                       ! total number of steps: 600s / tstep
+  statefreq         = 200                       ! number of steps between screen dumps
+  restartfreq       = -1                        ! don't write restart files if < 0
+  runtype           = 0                         ! 0 = new run
+  qsplit            = -1                        ! timesteps set via se_tstep, dt_remap_factor, dt_tracer_factor
+  rsplit            = -1
+  integration       = 'explicit'                ! explicit time integration
+  tstep_type        = 9                         ! 1 => default method
+!  tstep_type        = 5                         ! 1 => default method
+  tstep             = 0.1 !! 20x20x60lev 0.1                       ! dynamics timestep
+  dt_remap_factor   = 1				! remap every 1 time steps
+  dt_tracer_factor  = 1                         ! tracers run at dynamics time step
+  hypervis_scaling  = 3.0
+  nu                = 0.02 !!!!0.216784
+  nu_top            = 0.0             
```

Comment:
It would be nice to find a scaling and nu value that worked generically for all planar tests. I can work on that!

Reply:
I just now tried the default EAM value of 3.4e-8  and the bubble is rather noisy. if you have a suggestion I will appreciate that.

Reply:
How does the value used for the gravity wave tests (0.216784) work?